modules/extensions/src/views/lite/components/debugger/components/Predictions.tsx

```diff
@@ -23,22 +24,26 @@ const Predictions = (props: Props) => {
       <div className={style.sectionTitle}>Top Predictions</div>
       {Object.keys(predictions).map((key, index) => {
         const { confidence, intents } = predictions[key]
-        return (
-          <div className={style.subSection} key={index}>
-            <p>
-              {key} {formatConfidence(confidence)}%
-            </p>
-            <ul>
-              {intents.slice(0, 4).map(i => {
-                return (
-                  <li key={i.label}>
-                    <Intent name={i.label} confidence={i.confidence} />
-                  </li>
-                )
-              })}
-            </ul>
-          </div>
-        )
+        if ((key === 'global' && activePrompt) || key !== 'global') {
+          return (
+            <div className={style.subSection} key={index}>
+              <div>
+                <p>
+                  {key} {formatConfidence(confidence)}%
```

Comment:
I know you didn't write this piece of code but why not include the % in the formatConfidence function ?

Reply:
Changed